clang/lib/DPCT/RulesAsm/Parser/AsmNodes.h

```diff
@@ -222,9 +223,7 @@ class InlineAsmVarDecl : public InlineAsmNamedDecl {
       : InlineAsmNamedDecl(VariableDeclClass, Name), StateSpace(SS),
         Type(Type) {}
 
-  AsmStateSpace getStorageClass() const {
-    return StateSpace;
-  }
+  AsmStateSpace getStorageClass() const { return StateSpace; }
```

Comment:
Only format changes, can do it in a separate PR.

Reply:
Okay, I'm reverting this change back.